bin/resources/GameIndex.yaml

```diff
@@ -1540,7 +1546,7 @@ SCCS-40006:
   region: "NTSC-C"
 SCCS-40007:
   name: "Arc the Lad - Seirei no Tasogare"
-  region: "NTSC-C"
+  region: "NTSC-CH"
```

Comment:
this is wrong its NTSC-C ,  https://en.wikipedia.org/wiki/NTSC-C)

Reply:
Thanks, it wasn't clear because "NTSC-C" & "NTSC-CH" were both in the gamedb :)

Reply:
if their is any 'NTSC-CH' can you change the to 'NTSC-C' :slightly_smiling_face: